java/ql/test/query-tests/security/CWE-1104/semmle/tests/A.java

```diff
@@ -0,0 +1,2 @@
+public class A {
```

Comment:
Is this dummy file really needed?
[`security/CWE-548`](https://github.com/github/codeql/tree/main/java/ql/test/experimental/query-tests/security/CWE-548) does not have such a file either.

Reply:
Yes, without it, the database isn't built

Reply:
Do you know if `security/CWE-548` is broken then (or if not, why it works)?

Reply:
@aschackmull, do you know why `security/CWE-548` does not have a dummy Java file?

Reply:
Looks like a mistake in the `CWE-548` qltest. It also doesn't pass for me locally when explicitly run, so that is probably why CI didn't pick up the failure.

Reply:
Actually, I mixed things up - it does run successfully for me locally, but I'll add a dummy file for it in any case.

Reply:
https://github.com/github/codeql/pull/5195